### ISSUE TITLE
[Supporting the platform – Make Code - Language] 'Language' link controls does not adapt high contrast(Black/White) theme under 'Select Language' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3666]
**User Experience**:
Visual impaired user will find difficulty to see the control or element if that control does not adopt the high contrast mode, as they won't be able to take on that controls and they will miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Language " option and select it.
7. Navigate through all the controls appearing under "Select Language" dialog.
8. Verify that 'Language' link controls adapting high contrast theme or not.

**Actual Result:**
'Language' link controls does not adapt high contrast(Black/White) theme under 'Select Language' pop-up.

**Expected Result:**
'Language' link controls should adapt high contrast(Black/White) theme under 'Select language' pop-up. It should look as 'Yellow' color in high contrast black theme and 'Blue' color in high contrast white theme.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.3.1; #Win10-Edge(Chromium)

![WCAG4 3 1_Language_Normal Mode](https://user-images.githubusercontent.com/72871089/98908470-7e00ed00-24e6-11eb-9f26-01a086362c30.jpg)


![WCAG4 3 1_Language_HC Black Theme](https://user-images.githubusercontent.com/72871089/98908489-822d0a80-24e6-11eb-815e-d9c761e7e4e4.jpg)


![WCAG4 3 1_Language_HC White Theme](https://user-images.githubusercontent.com/72871089/98908501-8527fb00-24e6-11eb-9606-6b12e678ab92.jpg)

**Comment by shakao** [comment]
similar to https://github.com/microsoft/pxt-microbit/issues/3692, these buttons are not links and aren't used for page navigation. closing, since they should not have the link color
